### PR TITLE
VerticalResults: fix viewMore styling

### PR DESCRIPTION
**Pull request by oshi97**
per slack post w/Jeremy + Rose, the whole showMore div should be clickable,
and have background color #fafafa instead of white

TEST=manual
test that background color is #fafafa and whole div is clickable
and on hover text is underlined

### CODE REVIEW

File: src/ui/sass/modules/_Results.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 $results-title-bar-background: var(--yxt-color-background-highlight) !default;
 $results-filters-background: white !default;
+$results-view-more-background: var(--yxt-color-background-highlight);
```
Comment (creotutar) [inline src/ui/sass/modules/_Results.scss]: Can we add !default if an AEB user wanted to override them

Comment (oshi97) [reply src/ui/sass/modules/_Results.scss]: sorry can't believe I forgot this

File: src/ui/sass/modules/_Results.scss [hunk 2]
```diff
@@ -165,7 +167,8 @@ $results-cards-margin: calc(var(--yxt-base-spacing) / 2) !default;
     text-decoration: none;
     display: flex;
     align-items: center;
-    &:hover, &:focus 
+    &:hover .yxt-Results-viewAllLabel,
+    &:focus .yxt-Results-viewAllLabel
     {
       color: $color-brand-hover;
```
Comment (creotutar) [inline src/ui/sass/modules/_Results.scss]: this seems to change the text color too, is that what we want?

Comment (creotutar) [reply src/ui/sass/modules/_Results.scss]: by removing this, are we affecting legacy sites relying on this `&:hover` and `&:focus` (without viewAllLabel)?

Comment (oshi97) [reply src/ui/sass/modules/_Results.scss:173]: there are no legacy sites on this

Comment (oshi97) [reply src/ui/sass/modules/_Results.scss]: no legacy sites with this styling, this was newly added

Comment (oshi97) [reply src/ui/sass/modules/_Results.scss]: hmmmm the color is a good point though, it's not in the mock and it wasn't mentioned in the slack post
image
so I think we can just remove it and life will be much simpler

File: src/ui/templates/results/verticalresults.hbs [hunk 3]
```diff
@@ -71,11 +71,9 @@
 
 {{#*inline "viewAll"}}
   {{#if (and _config.isUniversal _config.viewMore _config.viewMoreLabel)}}
-    <div class="yxt-Results-viewAll">
-      <a class="yxt-Results-viewAllLink" href="{{ verticalURL }}">
-        <div class="yxt-Results-viewAllLabel">{{_config.viewMoreLabel}}</div>
-        <div data-component="IconComponent" data-opts='{ "iconName": "chevron" }'></div>
-      </a>
-    </div>
+    <a class="yxt-Results-viewAll yxt-Results-viewAllLink" href="{{ verticalURL }}">
```
Comment (creotutar) [inline src/ui/templates/results/verticalresults.hbs:74]: do we want to add check to see if there is a verticalURL before showing an empty `<a>` tag?

Comment (oshi97) [reply src/ui/templates/results/verticalresults.hbs:74]: updated!

**Review by creotutar — APPROVED** [review]
🥤 🥤 🥤 LGTM

File: src/ui/components/results/universalresultscomponent.js [hunk 4]
```diff
@@ -63,7 +63,7 @@ export default class UniversalResultsComponent extends Component {
       // Icon in the titlebar
       icon: config.sectionTitleIconName || config.sectionTitleIconUrl || 'star',
       // Url that links to the vertical search for this vertical.
-      verticalURL: config.url || verticalKey + '.html',
+      verticalURL: 'url' in config ? config.url : verticalKey + '.html',
```
Comment (creotutar) [inline src/ui/components/results/universalresultscomponent.js:66]: can we add this to README?

Comment (oshi97) [reply src/ui/components/results/universalresultscomponent.js:66]: cleaned up readme section!

**Pull request merged**